xml/obs_maintenance_setup.xml

```diff
@@ -48,20 +51,47 @@
   <itemizedlist>
    <listitem>
     <para> openSUSE:11.4 is the <xref linkend="obs.glos.ga_project"/> in
-     this example. Frozen and not changing anymore. </para>
+     this example. It is locked and not changing anymore. </para>
    </listitem>
    <listitem>
     <para> openSUSE:11.4:Update is the <xref
-      linkend="obs.glos.update_project"/> to release official updates.
-    </para>
+      linkend="obs.glos.update_project"/> to release official updates for the
+      locked openSUSE:11.4 project.
+      Thus it links to the openSUSE:11.4 project, inheriting all packages from there. </para>
    </listitem>
    <listitem>
     <para> openSUSE:Maintenance is the <xref
-      linkend="obs.glos.maintenance_project"/>. </para>
+      linkend="obs.glos.maintenance_project"/> which in this case maintains
+      the openSUSE:11.4:Update project (and optionally others as well). </para>
    </listitem>
    <listitem>
-    <para> openSUSE:Maintenance:IDxxx is the <xref
-      linkend="obs.glos.incident"/> project. </para>
+    <para> openSUSE:Maintenance:IDxxx is a <xref
+      linkend="obs.glos.incident"/> project created automatically by a maintenance
+      request. </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1>
+  <title>Project setup for the Maintenance Process</title>
+  <para>All workflow related projects must be set up with a proper project meta configuration.</para>
+  <itemizedlist>
+   <listitem>
+    <para> The <xref linkend="obs.glos.ga_project"/> must be locked by the project maintainer
+        by either using the <command>osc lock [PROJECT]</command> command</para>
```

Comment:
technically it is not required to be locked, but of course highly recommended as second line of defence (someone with write access did a mistake).